src/common/disk.rs

```diff
@@ -376,3 +432,71 @@ impl fmt::Display for DiskKind {
         })
     }
 }
+
+/// Used to determine what you want to refresh specifically on the [`Disk`] type.
+///
+/// ```no_run
+/// use sysinfo::{Disks, DiskRefreshKind};
+///
+/// let mut disks = Disks::new_with_refreshed_list_specifics(DiskRefreshKind::new());
+///
+/// for disk in disks.list() {
+///     assert_eq!(disk.total_space(), 0);
+/// }
+/// ```
+#[derive(Clone, Copy, Debug)]
+pub struct DiskRefreshKind {
+    kind: bool,
+    details: bool,
+    io_usage: bool,
+}
+
+impl Default for DiskRefreshKind {
+    fn default() -> Self {
+        Self {
+            kind: true,
+            details: false,
```

Comment:
I think people will be interested into this information by default. Can you switch it to `true` please?

Reply:
That will also make the code example of `DiskRefreshKind` not "lie" too. :laughing:

Reply:
Done.